kale/task.py

```diff
@@ -50,6 +60,9 @@ class Task(object):
     # this value.
     queue = None
 
+    # Pool to run tasks in.
+    _pool = None
```

Comment:
recommend calling this _task_process_pool or something slightly more descriptive.